maven-surefire-plugin/src/site/apt/examples/toolchains.apt.vm

```diff
@@ -36,6 +36,9 @@ Using Maven Toolchains with ${thisPlugin}.
     guide, ${thisPlugin} will launch the test jvm using the main toolchain
     configured in Maven.
 
+    Note that options like <<<forkCount>>> prevent the toolchains from being used.
+    See {{{https://issues.apache.org/jira/browse/SUREFIRE-720}}SUREFIRE-720} for more information.
+
```

Comment:
Instead of pointing issue - I would like to see description how `forkCount` option impact on toolchains.
Reading issue and all comments is not easy to find clue of problem.

Reply:
Isn't

> Note that options like <<<forkCount>>> prevent the toolchains from being used.

describing the impact?

I'm not familiar with the exact details why it is not working. If you expect that then someone else has to make the changes.

I'd think this is at least an improvement over the status quo, because currently it says that surefire picks up toolchains by default, while it doesn't if forkCount is set.